pkg/sqlite/load.go

```diff
@@ -40,7 +43,7 @@ func NewSQLLiteLoader(db *sql.DB, opts ...DbOption) (MigratableLoader, error) {
 		return nil, err
 	}
 
-	return &sqlLoader{db: db, migrator: migrator}, nil
+	return &sqlLoader{db: db, migrator: migrator, enableAlpha: &options.EnableAlpha}, nil
```

Comment:
I think it would keep more in line with the current apis if this were a DbOption you pass in:

`NewSqliteLoader(db, WithEnableAlpha(true))`

see db_options.go for other examples.